### PR TITLE
Support full tuningConfig by handling it in a more loosely typed way.

### LINKED ISSUE #116: Support full tuningConfig
This could potentially be done by making these more "untyped" in the base layer of the DruidBeams builder and just taking whatever the user provided.

### PR DESCRIPTION
**Pull request by gianm**
Fixes #116.

### CODE REVIEW

File: core/src/main/scala/com/metamx/tranquility/druid/DruidBeamMaker.scala [hunk 1]
```diff
@@ -88,15 +84,15 @@ class DruidBeamMaker[A: Timestamper](
           "format" -> "json",
           "timestampSpec" -> timestampSpec,
           "dimensionsSpec" -> rollup.dimensions.specMap
-        ).asJava
-      ).asJava,
-      "metricsSpec" -> rollup.aggregators.toArray,
+        )
+      ),
+      "metricsSpec" -> Jackson.parse[Seq[Dict]](druidObjectMapper.writeValueAsBytes(rollup.aggregators.toArray)),
```
Comment (jon-wei) [inline core/src/main/scala/com/metamx/tranquility/druid/DruidBeamMaker.scala]: What does this change affect? (What happens without the Jacksonparse and writeValueAsBytes?)

**Comment by jon-wei** [comment]
:+1: Looks good to me

File: core/src/main/scala/com/metamx/tranquility/druid/DruidBeamMaker.scala (hunk 1 above)
Comment (gianm) [inline core/src/main/scala/com/metamx/tranquility/druid/DruidBeamMaker.scala]: Without this, (I'm not sure why), Jackson doesn't correctly serialize the aggregator factories.

**Comment by fjy** [comment]
:+1:

**Pull request merged**